blockchain/testhelper/README.md

```diff
@@ -0,0 +1,16 @@
+testhelper
```

Comment:
We could also make this an `internal` package if we don't envision any external usage (hidden from default `godoc`, can't be imported outside the project).

Reply:
Addressed in the latest pushes. Made an internal package inside the package blockchain